compiler/z/codegen/ControlFlowEvaluator.cpp

```diff
@@ -169,6 +169,7 @@ virtualGuardHelper(TR::Node * node, TR::CodeGenerator * cg)
          case TR_DirectMethodGuard:
          case TR_NonoverriddenGuard:
          case TR_InterfaceGuard:
+         case TR_AbstractGuard:
          case TR_MethodEnterExitGuard:
          case TR_HCRGuard:
          //case TR_AbstractGuard:
```

Comment:
Can we either uncomment this one to use instead, or else delete it? The list on x86 is like this too

Reply:
Changed in https://github.com/eclipse/omr/pull/3473/commits/8617dd3b42d21960eb7135295da2fd32e956fe2c